nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/remote/RemoteCLICommand.java

```diff
@@ -171,52 +173,62 @@ public void fetchCommandModel() throws CommandException {
          */
         @Override
         protected boolean updateAuthentication() {
```

Comment:
Looking at the changes of this file and the one before there seems to be a common structure for the commands that maybe can be extracted so the code does not need to be "duplicated"?

Reply:
sUt#44f6uUn3

Reply:
I saw that but didn't include it as I'm not a fan of bringing refactors into bug fixes.